common/src/main/java/us/myles/ViaVersion/protocols/protocol1_11to1_10/chat/ChatMsgCharacterLimitAction.java

```diff
@@ -0,0 +1,55 @@
+package us.myles.ViaVersion.protocols.protocol1_11to1_10.chat;
+
+public enum ChatMsgCharacterLimitAction {
```

Comment:
It might be better to introduce the related interface and implement it in this enum rather than put logic to switch-statement.